### PR TITLE
Added Encoder Decoder

### PR DESCRIPTION
**Pull request by axiom24**

### CODE REVIEW

**Comment by MechCoder** [comment]
I think there is are whitespace errors. Do take care of those.

**Comment by debugger22** [comment]
which file?

File: BinPy/combi_logic/__init__.py [hunk 1]
```diff
@@ -1 +1,2 @@
-from mux_demux import MUX
\ No newline at end of file
+from mux_demux import MUX
```
Comment (MechCoder) [inline BinPy/combi_logic/__init__.py:1]: Here? I'm not sure though. I was just having a casual look.

Comment (debugger22) [inline BinPy/combi_logic/__init__.py:1]: Is newline important? I think it just gives warning.

Comment (MechCoder) [inline BinPy/combi_logic/__init__.py:1]: Are you using Sublime Text or something? If yes, then go to Preferences, Settings-Default, and change "ensure_newline_at_eof_on_save": False to 'True' . It worked for me.

Comment (MechCoder) [inline BinPy/combi_logic/__init__.py:1]: http://www.sublimetext.com/forum/viewtopic.php?f=3&t=12118

Comment (debugger22) [inline BinPy/combi_logic/__init__.py:1]: yea Sublime. Thanks!

**Pull request merged**